continuous_integration/environment-mindeps-optional.yaml

```diff
@@ -0,0 +1,54 @@
+name: test-environment
+channels:
+  - conda-forge
+dependencies:
+  # required dependencies
+  - packaging=20.0
+  - python=3.8
+  - pyyaml=5.3.1
+  - click=7.0
+  - cloudpickle=1.1.1
+  - partd=1.2.0
+  - fsspec=2021.09.0  # required for arrow I/O support
+  - importlib-metadata=4.13.0
+  - toolz=0.10.0  # required for min version of cytoolz
+  # optional dependencies pulled in by pip install dask[array,dataframe]
+  - numpy=1.21
+  - pandas=1.3
+  # optional dependencies used by dask
+  - bokeh=2.4.2
+  - cachey=0.1.1
+  - crick=0.0.3
+  - cytoolz=0.10.1
+  - fastavro=0.22.6
+  - fastparquet=0.7.0
+  - h5py=2.10.0
+  - ipycytoscape=1.0.1
+  - IPython=7.9.0
+  - jinja2=2.10.3
+  - lz4=3.1.1
+  - matplotlib=3.1.1
+  - mimesis=4.0.0
+  - psutil=5.6.3
```

Comment:
Similar comment to `tblib`, let's increase the minimum to `psutil=5.7.0` to match `distributed` 

https://github.com/dask/distributed/blob/1ccc312a3d240609a84e973f5bdc7d01eb042a22/pyproject.toml#L36